### ISSUE TITLE
Question about embedding using pretrained model of ECAPA and Xvectors

### ISSUE THREAD

**Issue by KAIMAN0** [speechbrain/speechbrain#840]
I am trying to use the pre-trained ECAPA and Xvectors model other than AMI and VoxCeleb. Should I normalize the embedding with mean_var_norm_emb during embed every speaker with multi-files? either ECAPA and Xvectors. Secondly, when doing the speaker recognition task of a single wave file, does it also need to normalize with mean_var_norm_emb? Many thanks!

**Comment by KAIMAN0** [comment]
Or should I set ["mean_var_norm_emb"].count = 1 for single file embedding?

**Comment by mravanelli** [comment]
Hi, I plan to refactor the recipe such that we store the embeddings while training them. In the current version, I save manually the normalization statistics in verification_cosine.py right after computing the training embeddings (used for s-normalization). In practice, you can process some data, store the normalization statistics with torch.save, and then use them for all the test files. You can find an example here: https://huggingface.co/speechbrain/spkrec-ecapa-voxceleb
